packages/attention/src/props.ts

```diff
@@ -93,4 +96,22 @@ export type AttentionProps = {
    * Handler that is called when the close button is clicked
    */
   onDismiss?: () => void;
+
+  /**
+   * The distance from which to offset the attentionEl from the targetEl vertically
+   */
+distance?: number;
```

Comment:
We really should add @warp-ds/eslint-config and husky integration to this repo soon 😅 There are some changes in this PR that made unnecessary diff noise. Maybe we could prioritize [a task for fixing it](https://nmp-jira.atlassian.net/browse/WARP-446) in the upcoming weeks! 🤞

Reply:
Agree! 👍